### DISCUSSION TITLE
Language % differences from original

### DISCUSSION THREAD

**Discussion by radbackup** [isar-community/isar#113] [Q&A] [answered]
I have noticed that Simon's Isar is 50% Dart and 44% Rust. However, for the community fork it's 71% Dart and 22% Rust. What happened?

**Comment by vicenterusso** [comment, marked as answer]
No idea. Clone both projects and use diff tools to check differences. They are essentialy the same project. Just remember to compare  correct community branch with correct original tags

**Reply by radbackup** [reply]
I ended up using Drift. It can't store custom objects directly, like Isar can. But with a little extra code it can store custom objects as a JSON string.
